### PR TITLE
Updates to examples - no schema change

### PR DESCRIPTION
**Pull request by nick-knowles**

### CODE REVIEW

**Comment by skinkie** [comment]
I guess this one is outdated, if the examples are not in master yet. Feel free to take the examples but absolutely not the rest.

File: NeTEx.spp [hunk 1]
```diff
@@ -739,6 +738,28 @@
 				<Folder FolderName="epip">
```
Comment (ue71603) [inline NeTEx.spp:738]: Remove NeTEx.spp from PR.

File: README.md [hunk 2]
```diff
@@ -3,6 +3,41 @@
 
 ## Core, Part 1 (Network),  Part 2 (Timetables), Part3 (Fares) Schemas
 
+Version 1.20 - Base version plus  further minor fixes comprising##
```
Comment (ue71603) [inline README.md:6]: Remove README.md from PR

File: xsd/NeTEx_publication.xsd [hunk 3]
```diff
@@ -16,28 +16,39 @@
 				<Contributor>CEN TC278 WG3 SG9 Team.</Contributor>
```
Comment (ue71603) [inline xsd/NeTEx_publication.xsd:16]: These changes are in my view already in one of the other PR. Here I think we can omit them. @nick-knowles or don't you agree?

File: xsd/netex_framework/netex_genericFramework/netex_organisation_version.xsd [hunk 4]
```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (ue71603) [inline xsd/netex_framework/netex_genericFramework/netex_organisation_version.xsd:1]: Also done in other PR. Can be removed here.

File: xsd/netex_part_2/part2_journeyTimes/netex_coupledJourney_version.xsd [hunk 5]
```diff
@@ -33,6 +33,9 @@
 				<Date>
```
Comment (ue71603) [inline xsd/netex_part_2/part2_journeyTimes/netex_coupledJourney_version.xsd:33]: In different PR, can be removed here.

File: xsd/netex_part_2/part2_journeyTimes/netex_flexibleService_version.xsd [hunk 6]
```diff
@@ -4,7 +4,8 @@
 	<xsd:include schemaLocation="netex_flexibleService_support.xsd"/>
```
Comment (ue71603) [inline xsd/netex_part_2/part2_journeyTimes/netex_flexibleService_version.xsd:4]: In different PR. Can be removed here.

**Review by ue71603 — CHANGES_REQUESTED** [review]
The XSD parts can be omitted. When then NeTEx.spp and REadme.md are removed from the PR, then I think we can try to merge it.

File: xsd/NeTEx_publication.xsd [hunk 7]
```diff
@@ -5519,6 +5519,26 @@ Correct COnstraints for PointOnRoute
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
+<!-- =====TypeOfLine unique========================== -->
```
Comment (skinkie) [inline xsd/NeTEx_publication.xsd:5522]: @ue71603 This looks that this is new, but is actually already in.

**Comment by skinkie** [comment]
Changes are already in current master.

**Pull request closed without merging**